src/object/schema_system_catalog_install.cpp

```diff
@@ -291,7 +290,26 @@ namespace cubschema
       static const system_catalog_definition &init_synonym ();
 
       // views
-      static const system_catalog_definition &init_vclass ();
+      static const system_catalog_definition &init_view_class ();
```

Comment:
get_data_of_view_... or get_def_of_view_... seems more suitable than init_view_... for the function